library/lua/gui.lua

```diff
@@ -916,6 +916,9 @@ end
 WINDOW_FRAME = make_frame('Window', true)
 PANEL_FRAME = make_frame('Panel', false)
 MEDIUM_FRAME = make_frame('Medium', false)
+MEDIUM_FRAME_2 = make_frame('Medium', false) --this is a panel style used in buildingplan's planner overlay
```

Comment:
```suggestion
MEDIUM_FRAME_2 = copyall(MEDIUM_FRAME)
```

Reply:
that's sick! totes gonna add that in as i commit the `INTERIOR_MEDIUM_FRAME` change